### PR TITLE
Enable processing crime container in prod (blob-router-service)

### PR DESCRIPTION
**Pull request by lgonczar**
### Change description ###

Enable processing crime container in prod (blob-router-service)

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: k8s/prod/common/reform-scan/blob-router.yaml [hunk 1]
```diff
@@ -21,6 +21,7 @@ spec:
       useInterpodAntiAffinity: true
       image: hmctspublic.azurecr.io/reform-scan/blob-router:prod-d9a7fbc1
       environment:
+        CRIME_ENABLED: true
```
Comment (lgonczar) [inline k8s/prod/common/reform-scan/blob-router.yaml]: Alternatively, we could just have crime enabled by default. Thoughts?

Comment (nitinprabhu) [reply k8s/prod/common/reform-scan/blob-router.yaml]: Crime processing won’t work on prod at the moment so better not to enable it for now ?

We could enable it by default in values yaml and disable for prod in flux ?

Comment (doncem) [reply k8s/prod/common/reform-scan/blob-router.yaml]: if not going to work on prod why are we enabling then here?

Comment (lgonczar) [reply k8s/prod/common/reform-scan/blob-router.yaml]: Why is it not going to work in prod?

**Pull request closed without merging**